src/lib.rs.in

```diff
@@ -173,7 +187,7 @@ impl Client {
     /// Take a nearly complete ElasticSearch URL, and stick the host/port part
     /// on the front.
     pub fn full_url(&self, suffix: &str) -> String {
-        format!("{}/{}", self.base_url, suffix)
+        format!("{}/{}", self.base_url.as_str(), suffix)
```

Comment:
I agree, but as users are not allowed to input their own paths (I mean, if I'm not missing anything only indexes are appended to `base_url`) it could be worth just `unwrap`ping the returned `Result`. I mean, if it happens to panic then we're doing something wrong.